src/posix/platform/daemon.cpp

```diff
@@ -74,15 +74,21 @@ void GetFilename(Filename &aFilename, const char *aPattern)
 
 int Daemon::OutputFormatV(const char *aFormat, va_list aArguments)
 {
-    char buf[OPENTHREAD_CONFIG_CLI_MAX_LINE_LENGTH + 1];
+    static constexpr char truncatedMsg[] = "(truncated ...)";
```

Comment:
Not sure if this is necessary. I don't see this design in syslog's RFC. https://www.rfc-editor.org/rfc/rfc5424.html

Reply:
Adding a notification message to the end of the output message helps users know what happens to the output message in the `ot-ctl`.